template/src/components/common/oauth/OAuthCallback.js

```diff
@@ -0,0 +1,8 @@
+/**
```

Comment:
Move all the files at `template/src/components/common/oauth/` to `template/src/lib/sdk/oauth`

Reply:
I'm aware we've redux dependencies that cannot be added as an external library straightforward. However, that will be pretty clear for the user they cannot modify those folders.

We'll figure out how we can package the sdk

With that in mind, if a user wants to upgrade, they can just update the lib folder.

Reply:
Ok, let's tackle that in 2 steps